src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -846,3 +846,16 @@ def validate_start_time(namespace):
     found = start_time_regex.findall(namespace.start_time)
     if not found:
         raise InvalidArgumentValueError('--start-time must be in format "HH:mm". For example, "09:30" and "17:00".')
+
+
+def validate_os_sku(namespace):
+    os_sku = namespace.os_sku
+    if os_sku in [CONST_OS_SKU_MARINER, CONST_OS_SKU_CBLMARINER]:
+        logger.warning(
+            'The options "%s" and "%s" will be renamed to %s, but "%s" and "%s" are still available.',
+            CONST_OS_SKU_MARINER,
+            CONST_OS_SKU_CBLMARINER,
+            CONST_OS_SKU_AZURELINUX,
+            CONST_OS_SKU_MARINER,
+            CONST_OS_SKU_CBLMARINER,
```

Comment:
```suggestion
            'The osSKU "%s" should be used going forward instead of "%s" or "%s". The osSKUs "%s" and "%s" will eventually be deprecated.',
            CONST_OS_SKU_AZURELINUX,
            CONST_OS_SKU_CBLMARINER,
            CONST_OS_SKU_MARINER,
            CONST_OS_SKU_CBLMARINER,
            CONST_OS_SKU_MARINER,
```

Reply:
Sure, updated.